### PR TITLE
Poster ratios

### PR DESCRIPTION
**Pull request by KubaTuta**
pictures sizes loading on resizing viewport

### CODE REVIEW

File: src/common/MovieTile/styled.js
Comment (HaniaNassalska) [inline src/common/MovieTile/styled.js:1]: There are still problems with sizes  of posters- some of them are smaller, also there is a problem with sapcing between the titles and the posters - there are differences. Acording to the project small tiles should not have space-between ;)

**Review by HaniaNassalska — COMMENTED** [review]
Besides movies tiles, everything working perfectly.
I know the problems with all the tiles in the project by heart - believe me ;) 
If you want, I can handle it :)

File: src/common/MovieTile/styled.js
Comment (KubaTuta) [reply src/common/MovieTile/styled.js:1]: put back old styles, with resizing they works just fine,
regarding space between I just think it looks cooler:)

**Review by HaniaNassalska — APPROVED** [review]
👌

**Pull request merged**